docs/api/testapi.html

```diff
@@ -2001,7 +2001,7 @@ <h5 id="_compat_args">compat_args</h5>
     my ($def_args, $fix_keys) = splice(@_, 0, 2);
     my %ret;
     for my $key (@{$fix_keys}) {
-        $ret{$key} = shift if (scalar(@_) &gt;= 1 &amp;&amp; (!defined($_[0]) || !grep(/^$_[0]$/, keys(%{$def_args}))));
+        $ret{$key} = shift if (scalar(@_) &gt;= 1 &amp;&amp; (!defined($_[0]) || !grep { $_ eq $_[0] } keys(%{$def_args})));
```

Comment:
I would not have expected this to appear in documentation. I think here and for some other subs implementation code is rendered into the documentation when it shouldn't. I will see if I can fix this